src/parser/mage/fire/modules/Checklist/Component.js

```diff
@@ -59,7 +59,8 @@ class FireMageChecklist extends React.PureComponent {
           {combatant.hasTalent(SPELLS.PHOENIX_FLAMES_TALENT.id) && <Requirement name="Phoenix Flames Charges" thresholds={thresholds.phoenixFlamesCombustionCharges} tooltip="When outside of Combustion, you should avoid using your Phoenix Flames charges so that they have time to come off cooldown before Combustion is available again. This will ensure that you have a couple charges so you can get as many Hot Streak procs as possible before Combustion ends. If you are about to cap on Phoenix Flames charges, then it is acceptable to use one." />}
           {combatant.hasTalent(SPELLS.FIRESTARTER_TALENT.id) && <Requirement name="Combustion during Firestarter" thresholds={thresholds.firestarterCombustionUsage} tooltip="If you are talented into Firestarter, you should ensure that you do not cast Combustion while the boss is above 90% Health. This would be a waste considering every spell is guaranteed to crit while the boss is above 90% Health, which defeats the purpose of using Combustion. Instead, you should use Combustion when the boss gets to 89% so you can continue the streak of guaranteed crits once Firestarter finishes." />}
           {combatant.hasTalent(SPELLS.FIRESTARTER_TALENT.id) && <Requirement name="Pyroclasm Usage" thresholds={thresholds.pyroclasmCombustionUsage} tooltip="If you have enough time to complete the cast before Combustion ends, then you should always use your Pyroclasm procs to hard cast Pyroblast during Combustion. This is primarily because Combustion will guarantee that the spell crits, resulting in more damage on top of the 225% that the Pyroclasm buff gives." />}
-          <Requirement name="Bad Spell Usage" thresholds={thresholds.combustionSpellUsage} tooltip="It is very important to use your time during Combustion wisely to get as many Hot Streak procs as possible before Combustion ends. To accomplish this, you should be stringing your guaranteed crit spells (Fireblast and Phoenix Flames) together to perpetually convert Heating Up to Hot Streak as many times as possible." />
+          <Requirement name="Bad Scorch Uses" thresholds={thresholds.scorchSpellUsageDuringCombustion} tooltip="It is very important to use your time during Combustion wisely to get as many Hot Streak procs as possible before Combustion ends. To accomplish this, you should be stringing your guaranteed crit spells (Fireblast and Phoenix Flames) together to perpetually convert Heating Up to Hot Streak as many times as possible. If you run out of instant spells, cast Scorch instead." />
+          <Requirement name="Bad Fireball Uses" thresholds={thresholds.fireballSpellUsageDuringCombustion} tooltip="Due to Combustion's short duration, you should never cast Fireball during Combustion. Instead, you should use your instant cast abilities like Fireblast and Phoenix Flames. If you run out of instant abilities, cast Scorch instead since it's cast time is shorter." />
```

Comment:
```suggestion
          <Requirement name="Bad Fireball Uses" thresholds={thresholds.fireballSpellUsageDuringCombustion} tooltip="Due to Combustion's short duration, you should never cast Fireball during it. Instead, you should use your instant cast abilities like Fireblast and Phoenix Flames. If you run out of instant abilities, cast Scorch instead because its cast time is shorter." />
```